### DISCUSSION TITLE
How to send logs to an already running prometheus server

### DISCUSSION THREAD

**Discussion by alikaz3mi** [prometheus/client_python#895] [Q&A]
Dear developers;

In the readme, you mentioned "To add Prometheus exposition to an existing HTTP server, see the MetricsHandler class which provides a BaseHTTPRequestHandler. It also serves as a simple example of how to write a custom endpoint."
I have difficulty implementing this. I am running a Prometheus docker on my machine on a dedicated port. How can I send logs to it? I couldn't find any example.

**Comment by csmarchbanks** [comment]
Hello, Prometheus does not support logs, are you looking for how to have Prometheus gather metrics from your service?
